services/QuillLMS/lib/tasks/users.rake

```diff
@@ -13,4 +13,22 @@ namespace :users do
       progress_bar.increment
     end
   end
-end
\ No newline at end of file
+
+  task refresh_school_subscriptions: :environment do
+    pipe_data = STDIN.read unless STDIN.tty?
+
+    unless pipe_data
+      puts 'No data detected on STDIN.  You must pass data to the task for it to run.  Example:'
+      puts '  rake users:refresh_school_subscriptions < path/to/local/file.csv'
+      puts ''
+      puts 'If you are piping data into Heroku, you need to include the --no-tty flag:'
+      puts '  heroku run rake schools:update_clever_ids -a empirical-grammar --no-tty < path/to/local/file.csv'
```

Comment:
I think this comment is a copy/paste error.

Reply:
Good catch.  I actually explicitly remember changing this text, but maybe I re-pasted it?  Will fix.